### ISSUE TITLE
Move the sync message for subscribed channels

### ISSUE THREAD

**Issue by a123qwertz567** [ram-on/SkyTube#817]
Every time I open the app it syncs my subscriptions, which is good. But The message lays over the tabs on the top (screenshot appended). This is annoying because if you swipe it away it keeps poping up for every channel.

Can you move the message to the bottom? Or replace it with a refresh/sync circle?

![Uploading Screenshot_20201016-230230_SkyTube.png…]()

**Comment by a123qwertz567** [comment]
![Uploading Screenshot_20201016-230230_SkyTube.png…]()

**Comment by bluesatoshi** [comment]
It couldn't be moved to the bottom because these are system notifications. They'd have to devise some method so it's not spitting out a new notification for each channel it checks.

**Comment by gzsombor** [comment]
Yeah, this notification thingy is nice, if you don't need to see for more than 10 secs, after that, it's really annoying.

**Comment by a123qwertz567** [comment]
Couldn't we just get an option to turn this off?

**Comment by gzsombor** [comment]
@codecyclist sure, improvements on the notification front is welcomed as well! Sorry for replaying late.
As I generally use a custom branch for my testing, which contains a lot of experimental, half finished feature, and I cherry-pick the changes when I think they are sort of working, and only test them together before a release, sometimes the master is not 100% working. But as it is rare that some one complains about it, I guess, no-one really want to use master :( 
I will have a look on it, what went wrong...

**Comment by drogga** [comment]
Those type of floating pop-up notifications (overlays) are called "Heads-Up Notifications" (samsung has it's own different name for those) and they can be easily disabled per-app basis or globally, either from the android's settings in most OEM's firmwares or by setting the "heads_up_notifications_enabled" value in Global Table to 0, or the easiest method is by just disabling the SkyTube's notifications completely, in case there isn't a priority (7.0 N+) options to set it to low/mute/silent, even tho I've never saw this app shows it like this even before I disabled those globally (because of "Aurora Store" and some other apps), instead it's a regular/normal low-priority "gentle" one, that isn't annoying at all, and there's a summary toast message when the fetching is finished with the total number of found new channel's uploads, so working on this would be a bit of waste of time, since there's basically no need, tho it can probably be tuned/tweaked up a bit to not re-appear when/once it's being dismissed, but that's a minor thing...

**Comment by navid-zamani** [comment]
I think a toast notification or the like generally is the wrong way, since it’s not persistent.

Instead, the app should mark all new videos in the feed list, so you can see what’s actually new. It currently has “not watched” and “watched” as states, but “not yet checked out” is missing. And it should not assume I have checked them out just because I had the feed tab open and they got listed once. I’d prefer to go over them and tap some checkbox. That would also allow me to mark a video as done without marking it as seen when I don’t like it and don’t want to watch it but want to see that I have checked it our before. That way I can easily scan the list for what’s new. Currently I can not distinguish between unwatched/new and unwatched/don’tWantTo. :)
